README.md

```diff
@@ -1,102 +1,35 @@
 # Fairphone Checkup
 
-An Android app that allows end users to test the features of their Fairphone 2.
+Hardware diagnostics app originally designed for the [Fairphone 2][fairphone2].
 
-If you are looking for build artifacts, please have a look at the **prebuilts** branch.
+## Summary
 
-## Versioning scheme
+Fairphone Checkup is an Android app that allows end users to test the features of their device.
+It is included in the Android OS for the Fairphone 2.
 
-The app version (a.k.a. version code) is an integer based on the following scheme:
+Since it is open source, the [WeAreFairphone community](wearefairphone) makes it available to users with other Android 5+ devices.
 
-* `<major version><minor version><revision>`
+You may want to take a look at Fairphone's [Recent Contacts][wearefairphone-mycontacts] and [Clock][wearefairphone-clock] widgets too.
```

Comment:
If you want, you may skip this paragraph (and links at the bottom) since they are not related with the app &mdash;contrary to launchers and widgets.